### PR TITLE
Added PropType package and changed from component to PureComponent

### PR DESCRIPTION
**Pull request by MHerszak**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "history": "^3.0.0",
     "html-to-text": "^2.1.0",
     "immutability-helper": "^2.0.0",
+    "import": "0.0.6",
```
Comment (xavcz) [inline package.json]: Is it needed?

File: package.json [hunk 2]
```diff
@@ -47,6 +48,7 @@
     "mingo": "^0.8.1",
     "moment": "^2.13.0",
     "optics-agent": "^1.0.5",
+    "prop-types": "^15.5.10",
     "react": "^15.4.1",
```
Comment (xavcz) [inline package.json]: Should we update React & React DOM also to the latest?

File: package.json (hunk 1 above)
Comment (MHerszak) [inline package.json]: @xavcz This import is not needed and I did not add it to the package.json. I merged with devel first and thought it came from there.

File: package.json (hunk 2 above)
Comment (MHerszak) [inline package.json]: @xavcz I'd say yes. There are no real "threats" until v16.

File: package.json (hunk 1 above)
Comment (xavcz) [inline package.json]: 👍

File: package.json (hunk 2 above)
Comment (xavcz) [inline package.json]: 👍

**Comment by SachaG** [comment]
Should I merge this? Or are you planning other changes?

**Comment by MHerszak** [comment]
@SachaG yes merge it. I will probably keep adding changes. Not today or tomorrow, though.

**Pull request merged**